examples/auth0/lib/auth0.js

```diff
@@ -1,15 +1,18 @@
 import { initAuth0 } from '@auth0/nextjs-auth0'
-import config from './config'
 
 export default initAuth0({
-  clientId: config.AUTH0_CLIENT_ID,
-  clientSecret: config.AUTH0_CLIENT_SECRET,
-  scope: config.AUTH0_SCOPE,
-  domain: config.AUTH0_DOMAIN,
-  redirectUri: config.REDIRECT_URI,
-  postLogoutRedirectUri: config.POST_LOGOUT_REDIRECT_URI,
+  clientId: process.env.NEXT_PUBLIC_AUTH0_CLIENT_ID,
+  clientSecret: process.env.AUTH0_CLIENT_SECRET,
+  scope: process.env.NEXT_PUBLIC_AUTH0_SCOPE || 'openid profile',
+  domain: process.env.NEXT_PUBLIC_AUTH0_DOMAIN,
+  redirectUri:
+    process.env.NEXT_PUBLIC_REDIRECT_URI ||
+    'http://localhost:3000/api/callback',
+  postLogoutRedirectUri:
+    process.env.NEXT_PUBLIC_POST_LOGOUT_REDIRECT_URI ||
+    'http://localhost:3000/',
   session: {
-    cookieSecret: config.SESSION_COOKIE_SECRET,
-    cookieLifetime: config.SESSION_COOKIE_LIFETIME,
+    cookieSecret: process.env.SESSION_COOKIE_SECRET,
+    cookieLifetime: 7200,
```

Comment:
Why is this one not an environment variable too like before?

Reply:
Also, looks like it was only defined in the server, because the config.js didn't export it in the client configs, not sure if that was intentional or a bug

Reply:
@lfades  , it was hard coded in the config file before and was not a env variable. I can make it env file.

I removed config file altogether and replaced it since config.js had a check for client and server side env variables which is handled by next.js now?

Reply:
What I mean is that the configs are now merged, before there was a file exporting a different object depending on the environment. I think the way you're doing is better. Did you test if the app works?

Reply:
yes, the app works ,i will make the changes suggested,

Reply:
@lfades  ,can you merge this if it looks good ,I can start working on other examples